autoPyTorch/api/base_task.py

```diff
@@ -701,6 +701,7 @@ def _search(
         precision: int = 32,
         disable_file_output: List = [],
         load_models: bool = True,
+        run_greedy_portfolio: bool = False
```

Comment:
Thanks for the enhancement:)
It looks good to me.
But can we make the argument more generic?
For example, it is likely to add another portfolio selection in the future, right?
So it would be better what kind of portfolio we build by an arg.
I would prefer `portfolio_selection='greedy'` rather than `run_greedy_portfolio`